Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM php:7.2
```

Comment:
In composer.json php pointed to minimal version 7.1

Reply:
Done.